tasks/main.yml

```diff
@@ -29,15 +29,18 @@
   register: uname_output
 
 - name: Install linux-image-extra-* packages to enable AuFS driver
-  apt: pkg={{ item }} state=present
+  apt:
+    pkg: "{{ item }}"
+    state: present
+    update_cache: yes
```

Comment:
Can you use cache valid time like here: https://github.com/rjduffner/docker.ubuntu/blob/bd874bd6c43090a062943e6b227c543747cac892/tasks/main.yml#L151